components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties

```diff
@@ -49,19 +45,17 @@ phenotips.PushPatient.male=Male
 phenotips.PushPatient.unknown=Unknown
 phenotips.PushPatient.negative=NO
 phenotips.PushPatient.oneServerMessage=There is only one configured remote server: 
-phenotips.PushPatient.patientUploadedMessage=This patient was uploaded to
-phenotips.PushPatient.patientUploadedLessThanHour=less than an hour ago
-phenotips.PushPatient.patientUploadedDays=days ago
-phenotips.PushPatient.patientUploadedHours=hours ago
-phenotips.PushPatient.patientUploadedAsID=and was assigned patient ID
+phenotips.PushPatient.patientUploadedLessThanHour=This patient was uploaded to __serverID__ less than an hour ago and was assigned patient ID __patientID__
+phenotips.PushPatient.patientUploadedDays=This patient was uploaded to __serverID__ __daysN__ day(s) ago and was assigned patient ID __patientID__
+phenotips.PushPatient.patientUploadedHours=This patient was uploaded to __serverID__ __hoursN__ hour(s) ago and was assigned patient ID __patientID__
 phenotips.PushPatient.patientUploadedUpdate=If data in the record has changed, push to __serverID__ again to update the record there.
 phenotips.PushPatient.updateRemotePatient=Update remote patient
 phenotips.PushPatient.createRemotePatient=Create new remote patient
 phenotips.PushPatient.loadingServers=Loading server list\u2026
 phenotips.PushPatient.serverDesc=Selected server does not allow updating existing patients, so can only push again to a new patient record.
 phenotips.PushPatient.successfulUpdate=Updated patient successfully.
-phenotips.PushPatient.successfulPush=Pushed patient successfully, ID of the new patient on the remote server is
-phenotips.PushPatient.pushFailedUpdatesDisabled=Unable to update this patient - updates are disbaled on the remote server
+phenotips.PushPatient.successfulPush=Pushed patient successfully, ID of the new patient on the remote server is __patientID__
```

Comment:
Here are other occurrences:
`phenotips.patientSharing.push.dialog.consents.warning=Consents that need to be granted before pushing could not be found. This might prevent you from pushing this patient to a remote server`
`phenotips.PushPatient.serverAndUserFirstMessage=Please login to the remote server first`
`phenotips.PushPatient.oneServerMessage=There is only one configured remote server: `
`phenotips.PushPatient.pushFailedUpdatesDisabled=Unable to update this patient - updates are disabled on the remote server`
`phenotips.PushPatient.pushFailedInvalidGuid=Unable to update this patient - stored remote GUID is incorrect. Maybe the patient was deleted on the remote server`